crates/revm/src/context/evm_context.rs

```diff
@@ -166,21 +166,6 @@ impl<DB: Database> EvmContext<DB> {
             return return_result(InstructionResult::CallTooDeep);
         }
 
-        let (account, _) = self
-            .inner
-            .journaled_state
-            .load_code(inputs.bytecode_address, &mut self.inner.db)?;
```

Comment:
This is slightly an edge case that is not a consensus error but would fix it to be consistent. CALL from instruction loads account that makes account warm loaded, while if we do it from tx as in `TransactTo` this is not the case, this is really a edge case where it is noticeable only that account will not be returned from finalization and only if tx is calling precompile.

We can add here `load_account` in place of `load_code`, something like `let _ = self...load_acccount(..)?`